airbyte-ci/connectors/metadata_service/lib/metadata_service/gcs_upload.py

```diff
@@ -304,6 +342,12 @@ def upload_metadata_to_gcs(bucket_name: str, metadata_file_path: Path, validator
 
     doc_version_uploaded, doc_version_blob_id = _doc_upload(metadata, bucket, docs_path, False, False)
     doc_inapp_version_uploaded, doc_inapp_version_blob_id = _doc_upload(metadata, bucket, docs_path, False, True)
+    manifest_yml_uploaded, manifest_yml_blob_id = upload_file_if_changed(
+        yaml_manifest_file_path, bucket, f"{METADATA_FOLDER}/{metadata.data.dockerRepository}/manifest.yaml"
+    )
+    components_py_uploaded, components_py_blob_id = upload_file_if_changed(
+        yaml_manifest_file_path, bucket, f"{METADATA_FOLDER}/{metadata.data.dockerRepository}/components.py"
+    )
```

Comment:
I'm not sure what "_doc_upload()" means - if "doc" means any document or a specific documentation artifact. Rather than wrapping in a new custom function, I just used the lower-level "upload_file_if_changed()" function that other functions seem to be wrapping.